README.md

```diff
@@ -502,6 +502,47 @@ export default Base.extend({
 });
 ```
 
+### Upgrading to 2.0
+
+Starting from Ember Simple Auth 2.0, authorizers and the session service's `authorize` will be deprecated. You will need to configure your data adapter to include authorization data in request headers. Refer to the [Ember Guides](https://guides.emberjs.com/v3.0.0/models/customizing-adapters/#toc_headers-customization) for details on header configuration.
```

Comment:
I'd maybe not make this paragraph so specific to Ember Data (as authorizers could be used without Ember Data as well). Also authorizers are deprecated before 2.0 already and will be removed from 2.0. So maybe something like this

_"Authorizers and the session service's `authorize` method are deprecated and will be removed from Ember Simple Auth 2.0. The concept seemed like a good idea in the early days of Ember Simple Auth but proved to provide little value for quite some added complexity. In order to replace authorizers in an application, simply get the session data from the session service and inject it where it's needed manually."_

_"In most cases, authorizers are currently used together with Ember Data adapters (refer to the [Ember Guides](https://guides.emberjs.com/v3.0.0/models/customizing-adapters/#toc_headers-customization) for details on adapters). Replacing authorizers in these scenarios is pretty straight forward:"_

(also, max 80 chars line length please 🙏 )